### ISSUE TITLE
Allow the cursor to move instead of scrolling

### ISSUE THREAD

**Issue by davidgiven** [davidgiven/wordgrinder#89]
It may be possible without too much effort to have the cursor move up and down the screen, rather than locking it in the centre. I should investigate to see if this works.

**Comment by semioticrobotic** [comment]
Would these potential changes affect the availability of the "locked" mode? The center-screen default cursor position is one of the many wordgrinder facets features I love.

**Comment by davidgiven** [comment]
I like it too (which is why I implemented it), so there would certainly be a switch.

**Comment by larrykollar** [comment]
It's possible. I forked an ancient console-mode outliner [https://github.com/larrykollar/tines](Tines) that has the option of locked or free-scrolling modes. I haven't touched that code, but feel free to examine/poach it.

**Comment by davidgiven** [comment]
I did actually attempt to implement this, and failed miserably. There's too much internal logic which assumes the cursor position is fixed. I'll probably have another go; it's required for anything mouse-related.

**Comment by kbarni** [comment]
I think this issue was solved by commit f4ab37331fa144c8cdc6ee77cb3042f8d88ca4c1 and bf97108eef19304eb49d806449b65397a2f9b96c, so it can be closed.

**Comment by bodnerry** [comment]
I could not figure out how to activate this feature. Is this something that should be written in the .wordgrinder/startup.lua file? If so, what needs to go in the file to enable the jump scroll mode?

This is desirable for people wanting to use wordgrinder with an e-ink display because keeping the edit line centered forces all lines to repeatedly refresh instead of occasionally when you reach the bottom of the window. thanks heaps.

**Comment by davidgiven** [comment]
Yes, jump scrolling got implemented. Look in File->Global Settings->Look and Feel in the current development version. It's turned off by default.
